packages/starter/example/components/ContextProvider.tsx

```diff
@@ -7,6 +7,7 @@ import { ConnectionProvider, WalletProvider } from '@solana/wallet-adapter-react
 import { WalletModalProvider as ReactUIWalletModalProvider } from '@solana/wallet-adapter-react-ui';
 import {
     GlowWalletAdapter,
+    NightlyWalletAdapter,
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.

Reply:
ok, removed